bin/travis/before_script.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+"$(dirname ${BASH_SOURCE[0]})/../../node_modules/.bin/sequelize" \
+  --config /etc/blocktogether/sequelize.json db:migrate
+"$(dirname ${BASH_SOURCE[0]})/../../run-dev.sh" \
+  > "$(dirname ${BASH_SOURCE[0]})/../../bt.log" 2>&1 &
```

Comment:
Because since it was in the background, the output was getting mixed in with the Casper output; this way we can `cat` it at the end if the test fails.